src/modules/tccpp/components/permissions-manager.ts

```diff
@@ -109,6 +109,23 @@ export default class PermissionManager extends BotComponent {
                 ],
             },
         };
+        const voice_permissions: permission_overwrites = {
+            ...default_permissions,
+            [this.wheatley.roles.no_voice.id]: no_interaction_at_all,
+            [this.wheatley.roles.no_off_topic.id]: no_interaction_at_all,
+        };
+        const member_voice_channel: permission_overwrites = {
+            ...voice_permissions,
+            [this.wheatley.guild.roles.everyone.id]: {
+                deny: [Discord.PermissionsBitField.Flags.ViewChannel],
+            },
+            [this.wheatley.roles.voice_deputy.id]: { allow: [Discord.PermissionsBitField.Flags.ViewChannel] },
+            ["331876085820030978"]: { allow: [Discord.PermissionsBitField.Flags.ViewChannel] }, // Intermediate
```

Comment:
```suggestion
            [this.wheatley.skill_roles.intermediate.id]: { allow: [Discord.PermissionsBitField.Flags.ViewChannel] },
```
etc would be good

Reply:
I should have noted: reason I didn't do this is because it would conflict with #148. But could do for now, just wanted to avoid adding new stuff that will need to be changed down the line.

Reply:
Discussed offline, will change so that we don't forget to fix later